src/components/queryExpressionEditor/QueryExpressionEditor.scss

```diff
@@ -0,0 +1,24 @@
+.container {
+    overflow: auto;
+}
+
+.leftContainer {
+    float: left;
+    width: 65%;
+}
+
+.rightContainer {
+    float: left;
+    width: 35%;
+}
+
+.topLeftContainer {
+    height: 114px;
+}
+
+.queryExpressionContainer {
+    height: 40px;
+    display: -webkit-flex;
+    display: flex;
```

Comment:
`.flex` and `.flex-center` in vapor can help you here https://github.com/coveo/react-vapor/pull/777/files